src/Calculator.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace Money;
+
+/**
+ * Interface Calculator
+ * @package Money
+ */
+interface Calculator
+{
+
+    /**
+     * @return bool
```

Comment:
I know it's kind of obvious, but can you please add some documentation here as well?